src/patternfly/components/DescriptionList/description-list.scss

```diff
@@ -81,6 +87,21 @@ $pf-c-description-list--breakpoint-map: build-breakpoint-map("base", "md", "lg",
   }
 }
 
+.pf-c-description-list__text {
+  &.pf-m-clickable {
```

Comment:
I wonder if "clickable" is too broad, and could conflict with term that's clickable in some other way but isn't underlined because  there isn't more info. The queue is more that it has more info than being interactive. I was thinking `.pf-m-help` since we use "help" in the table and form for the help icon popovers, but that could conflict with those icons if we add them. Maybe `.pf-m-help-text`? It should also make sense if we use it other places, which I imagine we might as an alternative to a bunch of help icons.